frontend/packages/console-app/src/components/access-modes/access-mode.tsx

```diff
@@ -36,6 +60,34 @@ export const AccessModeSelector: React.FC<AccessModeSelectorProps> = (props) =>
 
   const { t } = useTranslation();
 
+  const getPVCAccessModes = (resource: PersistentVolumeClaimKind, key: string) => {
+    const accessModeRadios = [
```

Comment:
Avoid introducing duplicate code. We can still translate these without copying them into each component. Create a function that returns these values and uses `i18next.t`